.github/pull_request_template.md

```diff
@@ -0,0 +1,40 @@
+Thank you for your contribution to Dynatrace's Configuration as Code tool `monaco`.\
+Before submitting this Pr, please make sure that:
+
+- [ ] Your code builds without any errors or warnings.
+- [ ] You added unit tests for your changes.
+- [ ] The commits are atomic and have a proper title and description (not only what changed but also why).
+- [ ] Public symbols have proper documentation.
+- [ ] You have installed your build and tested it manually.
+- [ ] Your PR description below contains 
+  1. a brief summary of what and why something changed,
+  2. user-visible changes (monaco users),
+  3. library changes,
+  4. examples, if applicable (for example, how the manifest/configs changed or how the library call now works).
+
+If this is your first PR, please have a look at our [Contributing Guidelines](https://github.com/Dynatrace/dynatrace-configuration-as-code/blob/main/CONTRIBUTING.md).
```

Comment:
For more information, please look [Contributing Guidelines](https://github.com/Dynatrace/dynatrace-configuration-as-code/blob/main/CONTRIBUTING.md).